src/Analyzer.Utilities/SyntaxGeneratorExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace Analyzer.Utilities
+{
+    public static class SyntaxGeneratorExtensions
```

Comment:
In the old code, the base fixer classes declared an abstract method named `GenerateOperatorDeclaration`, and the C#- and VB-specific fixer classes overrode that method to generate the correct syntax for each language. Now that the operator declarations are generated by extension methods rather than by derived classes, we don't get the automatic dispatch on language that we used to get. So the extension method `OperatorDeclaration` in this class takes a parameter `string language`, and explicitly dispatches on language _via_ a `switch` statement. It's the price we pay for DRYing out the code (but I'm open to suggestion for ways to eliminate the `switch`).